app/Http/Controllers/ProjectCollaboratorController.php

```diff
@@ -90,8 +93,11 @@ public function userStore(Request $request, User $user)
         $invitation = new ProjectCollaborator();
         $invitation->user_id = $user->id;
         $invitation->project_id = request()->project;
+        $invitation->is_project_owner = false;
         $invitation->from_collaborator = false;
         $invitation->skill_id = request()->skill;
+        //TODO add comment
+        // $invitation->invitation_message = request()->;
         $invitation->save();
```

Comment:
Some useful documentation on the matter: https://laravel.com/docs/5.3/eloquent-relationships#updating-many-to-many-relationships

This table seems wrong as it links user, project and skill together. What is the primary key? If `<user_id, project_id, skill_id>` is the primary key, then `is_project_owner` violates the 2nd normal form.